spec/integration/post_index_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'rails_helper'
+RSpec.describe 'Users/posts/ page', type: :system do
+  before do
+    @user1 = User.create(Name: 'Rodrigo De Paul',
+                         photo: 'https://upload.wikimedia.org/wikipedia/commons/thumb/0/09/Andrew_Carnegie%2C_by_Theodore_Marceau.jpg/800px-Andrew_Carnegie%2C_by_Theodore_Marceau.jpg',
+                         Bio: 'Industrialist.',
+                         PostsCounter: 2)
+    @first_post = Post.create(Title: 'Helloween', Text: 'This is my first post', user_id: @user1.id,
+                              comments_counter: 3, likes_counter: 2)
+    @second_post = Post.create(Title: 'Musically', Text: 'Micheal Jackson is the best ever', user_id: @user1.id,
+                               comments_counter: 3, likes_counter: 2)
+  end
+
```

Comment:
- [ ] You are missing the very first test case for post index page as per project requirements:
- Test that asserts you can see the user's profile picture.
Please do well to add this test case.

Reply:
resolve problem

Reply:
resolve problem